package.json

```diff
@@ -3,8 +3,7 @@
   "version": "1.0.0",
   "private": true,
   "engines": {
-    "node": ">=22.0.0",
-    "pnpm": "^9.0.6"
```

Comment:
What's the reason for removing the pinned pnpm version?

Reply:
newer version of the `pnpm` action didn't like having both defined, and Netlify only checks against `packageManager` so removed `pnpm` from the `engines` object